### PR TITLE
updated access policy for keyvault to allow jenkins access

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: csra/mock/azure-provider-config.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (glenjamin) [inline csra/mock/azure-provider-config.json]: should this be checked in?

File: csra/mock/csra.tf [hunk 2]
```diff
@@ -67,6 +67,12 @@ resource "azurerm_key_vault" "vault" {
         key_permissions = []
         secret_permissions = "${var.azure_secret_permissions_all}"
     }
+    access_policy {
+        tenant_id = "${var.azure_tenant_id}"
+        object_id = "${var.azure_jenkins_sp_oid}"
+        key_permissions = []
+        secret_permissions = "${var.azure_secret_permissions_all}"
```
Comment (glenjamin) [inline csra/mock/csra.tf]: maybe just read/write or maybe even write only?

Comment (mattops) [reply csra/mock/csra.tf]: Now updated to 'set' only, and this works.

File: azure-devtest.tf [hunk 3]
```diff
@@ -14,6 +14,10 @@ variable "azure_webops_group_oid" {
     type = "string"
     default = "98dc3307-f515-4717-b3c1-7174413e20b0"
 }
+variable "azure_jenkins_sp_oid" {
```
Comment (glenjamin) [inline azure-devtest.tf]: Is this the right TF config file?

**Comment by mattops** [comment]
New PR raised https://github.com/ministryofjustice/digital-studio-infra/pull/103

**Pull request closed without merging**